errors/src/iosMain/kotlin/dev/icerock/moko/errors/presenters/AlertErrorPresenter.kt

```diff
@@ -13,15 +12,15 @@ import platform.UIKit.UIAlertControllerStyleAlert
 import platform.UIKit.UIViewController
 
 actual class AlertErrorPresenter actual constructor(
-    private val exceptionMapper: (Throwable) -> StringDesc,
+    override val exceptionMapper: (Throwable) -> StringDesc,
     private val alertTitle: StringDesc,
     private val positiveButtonText: StringDesc
-) : ErrorPresenter {
+) : ErrorPresenter<StringDesc>() {
 
-    override fun show(exception: Throwable, viewController: UIViewController) {
+    override fun show(throwable: Throwable, viewController: UIViewController, data: StringDesc) {
         val alert = UIAlertController.alertControllerWithTitle(
             title = alertTitle.localized(),
-            message = exceptionMapper(exception).localized(),
+            message = exceptionMapper(throwable).localized(),
```

Comment:
use `data` instead of second `exceptionMapper` call

Reply:
Fixed.